runtime/object/bpf_object.cpp

```diff
@@ -125,14 +125,14 @@ bpftime_object::bpftime_object(std::string_view path)
 }
 
 // open the object elf file and load it into the context
-struct bpftime_object *bpftime_object_open(const char *obj_path)
+class bpftime_object *bpftime_object_open(const char *obj_path)
 {
-	struct bpftime_object *obj = new bpftime_object(obj_path);
+	class bpftime_object *obj = new bpftime_object(obj_path);
```

Comment:
why not just remove the struct keyword?

Reply:
My fault. I just replaced `struct bpftime_object` to `class bpftime_object` since clang telled me I shouldn't mix the usage of `struct bpftime_object` and `class bpftime_object`